regression-test/framework/src/main/groovy/org/apache/doris/regression/suite/Suite.groovy

```diff
@@ -601,6 +601,22 @@ class Suite implements GroovyInterceptable {
         }
     }
 
+    void checkTableData(String tbName1 = null, String tbName2 = null, int dataIndex = 0) {
+        def tb1Result = sql "select * FROM ${tbName1}"
+        def tb2Result = sql "select * FROM ${tbName2}"
```

Comment:
the result maybe different because random order, and decimal accuracy

Reply:
Thank you for your correction